spec/services/container_dashboard_service_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "spec_helper"
+
+describe ContainerDashboardService do
+  before(:each) do
+    ManageIQ::Providers::ContainerManager.delete_all
+  end
+
+  context "providers" do
+    it "returns all providers correctly" do
+      Tempfile.create(['config', 'yml']) do |f|
```

Comment:
> why is there a need to create a file during test? typically this won't run properly on all test envs. 
> iirc @simon3z also commented on this.

Its the same across the project like in ```spec/lib/vmdb/permission_stores_spec.rb```.


> iirc @simon3z also commented on this.

@simon3z  wrote: 
> @zeari for the record this is not optimal. Creating temp files should be the last resort. Vmdb::PermissionStore should allow you to load yaml from a string or a file object (not a file path).
Feel free to fix that if you have time ("not in this patch" ™)

So not in this patch.